### DISCUSSION TITLE
reuse components for multiple projects?

### DISCUSSION THREAD

**Discussion by mwildt** [DependencyTrack/dependency-track#3330] [Q&A]
We develop several applications that use a similar tech stack and reuse some components. If we now upload an SBOM in the dependency track for each of these projects, all shared components are created individually for each project - so we also have to maintain all vulnerabilities individually, even though they are actually the same component.

Is there any possibility that Dependency-Track does not create the components for each new project, but reuses existing components (e.g. based on the coordinates)? Or is it (even better) possible to create project groups in which the components are then only created once?

**Comment by felipesere** [comment]
I'm also interested in this. 
Particularly, I want to be able to answer: which projects use component X, and even component X@v1.2.3.

Is this something that DependencyTrack does? I haven't figured it out yet.
